src/test/data/JsonSerializableAddressBookTest/typicalAddressBook.json

```diff
@@ -63,16 +63,25 @@
     "maturityDate": "2021-04-06 07:00",
     "expiryDate": "2023-04-04 06:00"
   } ],
-  "orders": [ {
-    "orderId": 1,
+  "orders": [
+    {
+      "orderId": 1,
+      "cheeseType": "Brie",
+      "quantity": 1,
+      "orderDate": "2021-03-06 08:00",
+      "completedDate": null,
+      "customerId": 3,
+      "cheeseIds": []
+    },{
+    "orderId": 2,
     "cheeseType": "Camembert",
     "quantity": 1,
     "orderDate": "2021-02-05 07:00",
     "completedDate": "2022-02-10 13:00",
     "customerId": 1,
     "cheeseIds": []
   }, {
-    "orderId": 2,
+    "orderId": 3,
     "cheeseType": "Feta",
     "quantity": 5,
     "orderDate": "2021-03-06 08:00",
```

Comment:
Do the cheeses need to be updated to have the `isAssigned` attribute?

Reply:
Yeap but even if dont have it's by default false. 
Will add it in the next commit.